README.md

```diff
@@ -11,6 +11,30 @@ part of the array each recursive call considers.
 
 ## Runtime Analysis
 
-Analyse the time complexity of your implementation and give a $\Theta$ bound for
-its worst-case runtime. Add your answer, including your reasoning, to this
-markdown file.
+The outer loop starts with subarraySize = 1 and doubles it each time with (subarraySize *= 2).
+
+It continues until subarraySize reaches arrayLength. 
+
+It can be simplified to $2^n$.
+
+$2^0$ is our first iteration, equaling 1
+
+$2^1$ is our second iteration, equaling 2
+
+$2^2$ is our third iteration, equaling 4
+
+$2^n$ is our nth iteration, equaling n
+
+It can be simplified to $2^n$. We can turn this into $\Theta log_2(n)$. We start at 1, after each time we multiply it by two we get the next power of 2 for $n$
+
+For each time the outer loop runs, the inner loop does n/subarraySize work. subarraySize is doubling each time, I know that this is the geometric series so the inner loop can be expressed as $\Theta(n)$
+
+Accounting for the inner and outer loop, mergesort now is $\Theta nlog(n)$.
+
+mergeSubarrays() outer loop runs from leftIndex to rightIndex, covering all elements in the array which is $\Theta(n)$ elements. 
+
+If the elements are not in order the inner while activates, it has to shift elements to properly seat the value being held by rightIndex, which can cover $\Theta(n)$ elements on each iteration.
+
+From this I concluded that this implementation has a time complexity of $\Theta n^3log(n)$
```

Comment:
Not following you here -- how do you go from $n \log n$ to $n^3\log n$?

Reply:
The nested loops in $mergesort()$ make $nlogn$ calls to $mergesubarrays()$, and each call to $mergesubarrays()$ takes $n^2$ time

Reply:
Hmm, but shouldn't the normal mergesort be $n^2\log n$ then?

Reply:
Okay, I think I found my miscalculation. I came to $\Theta (n^2nlogn)$, and updated the readme accordingly.